src/openms/source/KERNEL/MSChromatogram.cpp

```diff
@@ -448,4 +454,4 @@ void MSChromatogram::mergePeaks(MSChromatogram& other)
   temp.reserve((this->end() - this->begin()) + (other.end() - other.begin()) );
   ContainerType::assign(temp.begin(), set_sum_similar_union(this->begin(), this->end(), other.begin(), other.end(), temp.begin()));
   this->setMetaValue("merged_with",String(other.getMZ()));   
```

Comment:
I would make this optional. There could be many merges and also setting metaValues is super expensive

Reply:
Added.